sync/class.jetpack-sync-defaults.php

```diff
@@ -257,12 +257,22 @@ static function is_whitelisted_option( $option ) {
 		return false;
 	}
 
+	static function get_max_sync_execution_time() {
+		$max_exec_time = intval( ini_get( 'max_execution_time' ) );
+		if ( $max_exec_time === 0 ) {
+			// 0 actually means "unlimited", but let's not treat it that way
```

Comment:
The indentation is off here. We're using a tab for this line, but spaces for the next line that starts with `$max_exec_time`.